packages/tools/fluid-runner/src/logger/fileLogger.ts

```diff
@@ -0,0 +1,120 @@
+/*!
```

Comment:
maybe loggerUtils.cs? it fills like this file has not much to do with file logger itself